docs/extras/autorcm.md

```diff
@@ -1,33 +1,39 @@
+	AutoRCM causes the console to believe it is bricked, and will automatically launch RCM upon boot for recovery purposes, without needing a jig. As RCM is a recovery mode from repair specialists, this is an intended feature from the device developer, though is also considered a softbrick. If you aren't careful, misuse of AutoRCM can lead to real **damage**, especially with units that cannot inject custom RCM payloads (like Mariko hardware). Please take care when using it. Please keep in mind that the console can no longer boot on its own, so you'll need a PC, phone, or other payload injector to start the console after a coldboot.
 
-	Note: Before you freak out if the switch's screen stays black when you power on your switch after activating AutoRCM, it's not bricked. it's in RCM.
+	Note: If the console screen stays black when pressing the power button after activating AutoRCM, remember that it's not broken. It's in RCM.
 
-!!! warning "If you don't have a boot0/1 backup yet..."
-	You really want to kill your console huh? If you haven't made a boot0/1 backup yet, it is recommended to make one **right now**.
+!!! warning "If you don't have a BOOT0/1 backup yet..."
+	You really want to kill your console, huh? If you haven't made a BOOT0/1 backup yet, it is recommended to make one **right now**.
 
 	1. Boot Hekate
 	2. Go to `Tools`, and select `Backup eMMC`
 	3. Then tap `eMMC BOOT0 & BOOT1` and let the process complete.
-   
-!!! danger "Disadvantages of AutoRCM" 
-	There are a couple of disadvantages you should consider before installing AutoRCM:
 
-	- Your Switch will not boot normally with AutoRCM installed. This means you will need to keep a device ready to inject a payload every time you power it on (similar to a "tethered jailbreak"). Note: sleep mode is not considered "powering on" the console. Sleep mode will still work as intended
-	- When completely discharged, your Switch will take a very long time to charge while in RCM. This means that you'll be waiting around an hour or two for the battery to fully recharge to even boot your console again.
+!!! danger "Disadvantages of AutoRCM"
+	There are some disadvantages you should consider before installing AutoRCM:
+
+	- Please keep in mind that the console can no longer boot on its own, so you'll need a PC, phone, or other payload injector to start the console after letting the console fully shut off. (AutoRCM by itself is considered a "tethered jailbreak").
+		Note: Sleep Mode is not considered "powering off/on" the console. Sleep Mode will still work as intended and is fully unchanged with AutoRCM.
+	- When completely discharged, your Switch will take a very long time to charge while in RCM. To solve this, charge the console for approximately twenty minutes before booting into Hekate and selecting a boot option. After it's done booting, the console will now charge at a normal speed with a battery icon in the corner.
```

Comment:
Pretty sure that _just_ booting Hekate is enough to kickstart the charging process properly, and you can do this much earlier before booting HOS (from what I know, HOS tends to power off if there's less than 10% active charge). Might be useful to change this to reflect that.